### PR TITLE
DSP: Print hash of firmware to console

### PR DESCRIPTION
**Pull request by MerryMage**
It would be useful to know what version of the DSP firmware is used by an application when receiving a bug report.

### CODE REVIEW

File: src/core/hle/service/dsp_dsp.cpp [hunk 1]
```diff
@@ -77,18 +80,25 @@ static void LoadComponent(Service::Interface* self) {
...
     u32 size       = cmd_buff[1];
-    u32 unk1       = cmd_buff[2];
-    u32 unk2       = cmd_buff[3];
-    u32 new_size   = cmd_buff[4];
+    u32 prog_mask  = cmd_buff[2];
+    u32 data_mask  = cmd_buff[3];
     u32 buffer     = cmd_buff[5];
 
-    cmd_buff[1] = 0; // No error
+    cmd_buff[0] = 0x00110082;
```
Comment (Subv) [inline src/core/hle/service/dsp_dsp.cpp]: Please use IPC::MakeHeader

Comment (MerryMage) [inline src/core/hle/service/dsp_dsp.cpp]: Thanks! Corrected.

**Comment by Subv** [comment]
LGTM

**Comment by yuriks** [comment]
LGTM

**Pull request merged**